File: C#/백준/Bronze/13670. Alarme Despertador/Alarme Despertador.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            while(true){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int a = aa[0]; int b = aa[1]; int c = aa[2]; int d = aa[3];
                if(a==b && b==c && c==d && d==0){
                    break;
                }
                int t = c*60+d-(a*60+b);
                t = t<0?t+1440:t;
                Console.WriteLine(t);
            }
        }
    }
}

File: Python/백준/Bronze/13670. Alarme Despertador/Alarme Despertador.py
while(True):
    a,b,c,d = map(int,input().split())
    if a==b and b==c and c==d and d==0:
        break
    t = c*60+d-(a*60+b)
    if t<0:
        t+=1440
    print(t)